search/api_test.py

```diff
@@ -191,6 +214,83 @@ def test_index_users(self):
         index_users(User.objects.iterator(), chunk_size=4)
         assert_search(search(), list(User.objects.all()))
 
+    def test_add_certificate(self):
+        """
+        Test that Certificate is indexed after being added
+        """
+        user = UserFactory.create()
+        assert search()['total'] == 1
+        CachedCertificateFactory.create(user=user)
+        assert_search(search(), [user])
+
+    def test_update_certificate(self):
+        """
+        Test that Certificate is reindexed after being updated
+        """
+        user = UserFactory.create()
+        assert search()['total'] == 1
+        certificate = CachedCertificateFactory.create(user=user)
+        certificate.data = [{'new': 'data'}]
```

Comment:
need to check: I did not write these tests but I want to be sure there is a reason